### ISSUE TITLE
MemberExpression crashes SimpleExpressionVisitor if there is no (grand) parent InvocationExpression

### ISSUE THREAD

**Issue by itzstatic** [streamx-co/ExTree#2]
`class Entity { public int id; }`

```
Function<Entity, Integer> lambda = (Serializable & Function<Entity, Integer>)e -> e.id;
SimpleExpressionVisitor visitor = new SimpleExpressionVisitor(){ };
LambdaExpression.parse(lambda).accept(visitor);
```

Crashes with Stack Index Out Of Bounds when visiting MemberExpression's instance. Crash does *not* happen if there is an InvocationExpression somewhere in the tree, like if we have locally enclosed variables:

```
int x = 2;
Function<Entity, Integer> lambda = (Serializable & Function<Entity, Integer>)e -> e.id + x;
```
does not crash

**Comment by kostat** [comment]
The issue is fixed in 776ea6724b23bf171dcb0582b63bfb83483bc940. Do you need an artifact urgently or it can wait until next iteration?

**Comment by kostat** [comment]
@itzstatic, please let me know if you are waiting for an artefact with the fix
